packages/design-system/src/TableDropdown/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { Position } from "@blueprintjs/core/lib/esm/common/position";
+import {
+  Popover,
+  PopoverInteractionKind,
+} from "@blueprintjs/core/lib/esm/components/popover/popover";
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import Icon, { IconSize } from "Icon";
+import Spinner from "Spinner";
+import Text, { TextType } from "Text";
+import { CommonComponentProps } from "Types/common";
+import { Classes } from "Constants/classes";
+
+type DropdownOption = {
+  name: string;
+  desc: string;
+};
+
+export type DropdownProps = CommonComponentProps & {
```

Comment:
exporting this will clash with the one in Dropdown I think

Reply:
Removed export